addon/pods/components/frost-list/component.js

```diff
@@ -70,6 +69,14 @@ export default Ember.Component.extend({
     })
   }),
 
+  //  Optional attrs for sm vertical-collection
+  //  https://github.com/runspired/smoke-and-mirrors/blob/develop/addon/components/vertical-collection.js
+  idForFirstItem: null,
+  scrollPosition: 0,
+  key: '@identity',
```

Comment:
This is used together with idForFirstItem to let SM scroll to specific item(with specific key value) on initialization. @identity is the default value  {{vertical-collection}} takes which mains looking for 'id' attribute of DOM. It's the same as key: 'id'.  idForFirstItem, scrollPosition, key are currently not used in list, but they could be useful when we want to add more features.